pylabrobot/pumps/calibration.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+import json
+import csv
+from typing import Union, Dict, List, Optional
+
+
+class PumpCalibration:
+  """ Calibration for a single pump or pump array
+
+  Attributes:
+      calibration: The calibration of the pump or pump array.
+
+  """
+
+  def __init__(self, calibration: Optional[List[Union[float, int]]] = None):
+    """
+    Args:
+        calibration: calibration of the pump in pump-specific volume per time/revolution units.
+    Raises:
+        ValueError: if a value in the calibration is outside expected parameters.
+    """
+    if calibration is not None and any(value <= 0 for value in calibration):
+      raise ValueError("A value in the calibration is is outside expected parameters.")
```

Comment:
Perhaps specify what the expected parameters are in the doc string / error message?

Reply:
made this change!